kitsune/users/utils.py

```diff
@@ -87,8 +88,9 @@ def anonymize_user(user):
     # Deactivate the user and change key information
     user.username = "user%s" % user.id
     user.email = "%s@example.com" % user.id
+    # Delete the inbox of the user
+    InboxMessage.objects.filter(to=user).delete()
```

Comment:
Does the user's outbox also need to be deleted?

Reply:
No, deleting your own account shouldn't affect the recipients inbox.